src/main/java/com/eucalyptus/tests/awssdk/TestRoute53IamPolicy.groovy

```diff
@@ -0,0 +1,364 @@
+                      action: 'CREATE',
+                      resourceRecordSet: new ResourceRecordSet(
+                          name: 'name.change1.example.com.',
+                          type: 'A',
+                          TTL: 600,
+                          resourceRecords: [
+                              new ResourceRecord(value: '1.1.1.1')
+                          ]
+                      )
+                  )
+              ]
+          )
+      )).with {
+        Assert.assertNotNull('ChangeInfo', changeInfo)
+        Assert.assertEquals('ChangeInfo.comment',
+            'Change resource records comment here', changeInfo.comment)
+      }
+      N4j.print( "Deleting resource record set for public hosted zone ${hostedZoneId1}" )
+      changeResourceRecordSets(new ChangeResourceRecordSetsRequest(
+          hostedZoneId: hostedZoneId1,
+          changeBatch: new ChangeBatch(
+              comment: 'Change resource records comment here',
+              changes: [
+                  new Change(
+                      action: 'DELETE',
+                      resourceRecordSet: new ResourceRecordSet(
+                          name: 'name.change1.example.com.',
+                          type: 'A',
+                          TTL: 600,
+                          resourceRecords: [
+                              new ResourceRecord(value: '1.1.1.1')
+                          ]
+                      )
+                  )
+              ]
+          )
+      )).with {
+        Assert.assertNotNull('ChangeInfo', changeInfo)
+        Assert.assertEquals('ChangeInfo.comment',
+            'Change resource records comment here', changeInfo.comment)
+      }
+      null
+    }
+
+    try {
+      getRoute53Client(testAcctZoneUserCredentials).with {
+        N4j.print( "Changing resource records for public hosted zone ${hostedZoneId2}" )
+        changeResourceRecordSets(new ChangeResourceRecordSetsRequest(
+            hostedZoneId: hostedZoneId2,
+            changeBatch: new ChangeBatch(
+                comment: 'Change resource records comment here',
+                changes: [
+                    new Change(
+                        action: 'CREATE',
+                        resourceRecordSet: new ResourceRecordSet(
+                            name: 'name.change.example.com.',
+                            type: 'A',
+                            TTL: 600,
+                            resourceRecords: [
+                                new ResourceRecord(value: '1.1.1.1')
+                            ]
+                        )
+                    )
+                ]
+            )
+        )).with {
+          Assert.assertNotNull('ChangeInfo', changeInfo)
+          Assert.assertEquals('ChangeInfo.comment',
+              'Change resource records comment here', changeInfo.comment)
+        }
+        null
+      }
+      Assert.fail("Expected failure when updating hosted zone without permission")
```

Comment:
I guess I'm misreading the test: should the testAcctAoneUserCreentials be good to create a rrset?

Reply:
This is testing that the resource is checked, it should fail as the user does not have permission to access hostedZoneId2